rootfs/strap.py

```
@@ -51,12 +52,16 @@
-        package = parts[1]
+        package = re.sub(r'[^a-zA-Z0-9._+\-]', '', os.path.basename(parts[1]))
+        if not package:
+            print("  ==> Error: invalid package name")
+            exit(1)
         print(f"=> Uninstalling {package}")
 
         # read the save file
         save = ""
-        with open(f"rootfs/filesystem/etc/car/saves/{package}", "r") as f:
+        safe_save_path = os.path.join("rootfs/filesystem/etc/car/saves", package)
+        with open(safe_save_path, "r") as f:
```

Ruff (0.15.17)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)